examples/starlark-inject-sidecar/.expected/diff.patch

```diff
@@ -1,13 +1,10 @@
 diff --git a/deployment.yaml b/deployment.yaml
-index 007ecd3..04fda79 100644
+index 007ecd3..d691d7d 100644
 --- a/deployment.yaml
 +++ b/deployment.yaml
-@@ -15,6 +15,8 @@ spec:
-         app: nginx
-     spec:
-       containers:
-+        - name: logging-agent
-+          image: k8s.gcr.io/fluentd-gcp:1.30
-         - name: nginx
+@@ -19,3 +19,5 @@ spec:
            image: nginx:1.14.2
            ports:
+             - containerPort: 80
++        - image: k8s.gcr.io/fluentd-gcp:1.30
```

Comment:
This seems to be an issue. Ideally, user expects name to be first. Go functions can decide the order while generation but seems like starlark and ts functions can't.

cc @droot

Reply:
https://github.com/GoogleContainerTools/kpt/issues/2405

Reply:
Ack.